lib/at_host/at_host.c

```diff
@@ -36,6 +37,8 @@ K_THREAD_STACK_DEFINE(at_host_stack_area, AT_HOST_STACK_SIZE);
 #define AT_BUF_SIZE CONFIG_AT_CMD_RESPONSE_MAX_LEN
 #endif
 
+AT_MONITOR(at_host, NULL, response_handler);
```

Comment:
I guess we should use `ANY` instead of `NULL` to really use the at_monitor.h API definitions.